src/background/mobileEvents.js

```diff
@@ -0,0 +1,36 @@
+import {Component} from "./component.js";
+import {constants} from "./constants.js";
+
+export class MobileEvents extends Component {
+  async init() {
+    if (!constants.isAndroid) {
```

Comment:
Will `constants.isAndroid` be defined yet, or do we need to explicitly call `await constants.init();` first?
Or does it only need to be initialized once, which is already done in src/background/main.js above?